shopinvader_sale_packaging/models/shopinvader_variant.py

```diff
@@ -47,4 +47,14 @@ def _get_variant_packaging_ctx(self):
             "lang": self.lang_id.code,
             # consider only packaging that can be sold
             "_packaging_filter": lambda x: x.can_be_sold,
+            "_packaging_values_handler": self._prepare_qty_by_packaging_values,
+        }
+
+    def _prepare_qty_by_packaging_values(self, packaging, qty_per_pkg):
+        return {
+            "id": packaging.id,
+            "qty": qty_per_pkg,
```

Comment:
Why not `packaging.qty` here?

Reply:
because it's not what we want :)
`qty_per_pkg` is the qty computed by stock_packaging_calculator and it's based on contained packaging when computed for contained packaging. 
Indeed, above, I pass pkg.qty explicitly because I'm reusing the same method for both cases.